cmd/basecli/commands/cmds.go

```diff
@@ -76,6 +83,16 @@ func doSendTx(cmd *cobra.Command, args []string) error {
 	return txcmd.OutputTx(bres)
 }
 
+// WrapChainTx will wrap the tx with a ChainTx from the standard flags
+func WrapChainTx(tx basecoin.Tx) (res basecoin.Tx, err error) {
+	expires := viper.GetInt64(FlagExpires)
```

Comment:
just use uint as well? or is there a reason to load it in as an signed int and check here?